### PR TITLE
2020 - add capyweb as sponsor (silver and diversity) and fix sponsor background

### PR DESCRIPTION
**Pull request by lucascavalcante**

### CODE REVIEW

**Review by alessandrofeitoza — DISMISSED** [review]

File: 2020/en/index.html [hunk 1]
```diff
@@ -252,7 +252,7 @@ <h5 class="description">The partnership is strong, the support is sensational. T
                     <div class="col-md-4">
                         <div class="card card-pricing">
                             <div class="card-body">
-                                <h4 class="card-category"><strong>Gold Sponsor</strong></h4>
+                                <h4 class="card-category"><strong>Sponsor Gold</strong></h4>
```
Comment (cgrandval) [inline 2020/en/index.html]: Can you please keep Gold Sponsor :-)

File: 2020/en/index.html [hunk 2]
```diff
@@ -266,6 +266,24 @@ <h3 class="card-title">Blackfire.io</h3>
                         </div>
                     </div>
                 </div>
+                <div class="row justify-content-md-center">
+                    <div class="col-md-4">
+                        <div class="card card-pricing" style="background-color: #161F35;">
+                            <div class="card-body">
+                                <h4 class="card-category text-white"><strong>Sponsor Silver</strong></h4>
```
Comment (cgrandval) [inline 2020/en/index.html]:
```suggestion
                                <h4 class="card-category text-white"><strong>Silver Sponsor</strong></h4>
```

File: 2020/en/sponsors.html [hunk 3]
```diff
@@ -170,23 +170,25 @@ <h5 class="description">The partnership is strong, the support is sensational. T
                 <div class="space-top"></div>
                 <div class="row">
                     <div class="col-md-12">
-                        <h3 class="text-dark">Sponsors Gold</h3>
+                        <h3 class="text-dark">Sponsor Gold</h3>
```
Comment (cgrandval) [inline 2020/en/sponsors.html]: You can keep `Sponsors` as we can have 3, but you can change to `Gold Sponsors`

File: 2020/en/sponsors.html [hunk 4]
```diff
@@ -199,15 +201,59 @@ <h3 class="card-title">Blackfire.io</h3>
                     </div>
                 </div>
                 <div class="space-top"></div>
+                <div class="row">
+                    <div class="col-md-12">
+                        <h3 class="text-dark">Sponsor Silver</h3>
```
Comment (cgrandval) [inline 2020/en/sponsors.html]: Silver Sponsors

File: 2020/en/sponsors.html [hunk 5]
```diff
@@ -199,15 +201,59 @@ <h3 class="card-title">Blackfire.io</h3>
...
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-4">
+                        <h3 class="text-dark">Sponsor Diversity</h3>
+                        <br />
+                        <div class="card" style="background-color: #161F35;">
+                            <div class="card-body">
+                                <div class="card-icon" style="min-height: 300px;">
+                                    <img src="assets/img/sponsors/capyweb_logo.png" alt="Blackfire" style="margin-top: 30px;">
```
Comment (cgrandval) [inline 2020/en/sponsors.html]:
```suggestion
                                    <img src="assets/img/sponsors/capyweb_logo.png" alt="Capyweb" style="margin-top: 30px;">
```

File: 2020/en/sponsors.html [hunk 6]
```diff
@@ -199,15 +201,59 @@ <h3 class="card-title">Blackfire.io</h3>
...
+                        <div class="card">
+                            <div class="card-body">
+                                <div class="card-icon" style="min-height: 300px;">
+                                    <img src="assets/img/sponsors/blackfire.png" alt="Blackfire" style="width: 40%; margin-top: 70px;">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-4">
+                        <h3 class="text-dark">Sponsor Diversity</h3>
```
Comment (cgrandval) [inline 2020/en/sponsors.html]:
```suggestion
                        <h3 class="text-dark">Diversity Sponsors</h3>
```

File: 2020/sponsors.html [hunk 7]
```diff
@@ -197,16 +199,64 @@ <h3 class="card-title">Blackfire.io</h3>
...
+                        <div class="card">
+                            <div class="card-body">
+                                <div class="card-icon" style="min-height: 300px;">
+                                    <img src="assets/img/sponsors/blackfire.png" alt="Blackfire" style="width: 40%; margin-top: 70px;">
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-4">
+                        <h3 class="text-dark">Patrocinador Diversidade</h3>
```
Comment (cgrandval) [inline 2020/sponsors.html]:
```suggestion
                        <h3 class="text-dark">Patrocinadores Diversidade</h3>
```

**Review by cgrandval — CHANGES_REQUESTED** [review]

**Review by cgrandval — APPROVED** [review]

**Pull request merged**